code/game/area/areas/ruins/lavaland.dm

```diff
@@ -37,6 +37,12 @@
 	icon_state = "dk_yellow"
 	ambientsounds = HIGHSEC
 
+/area/ruin/powered/kinggoat_arena //yogs start
```

Comment:
Can you dunk this into ``/yogstation/code/game/area/areas/ruins/lavaland.dm``?

Reply:
no cause then it complains a ton about duplicate defs

Reply:
Why would it do that...?

Reply:
heck if I know